llms/ollama/options.go

```diff
@@ -34,6 +35,19 @@ func WithFormat(format string) Option {
 	}
 }
 
+// WithKeepAlive controls how long the model will stay loaded into memory following the request (default: 5m)
+// only supported by ollama v0.1.23 and later
+//
+//	If set to a positive duration (e.g. 20m, 1hr or 30), the model will stay loaded for the provided duration
```

Comment:
Thank you very much for your code, it solved my urgent problem! However, I have a small suggestion - there's a slight misunderstanding with the unit for "hours" in the comments. It should be "h" instead of "hr". When input hr, it will got an error message like this: time: unknown unit "hr" in duration "6hr".

Reply:
ok, it's copied from ollama docs, I will fix it.